scss/_buttons.scss

```diff
@@ -92,7 +96,8 @@
     &,
     &:hover {
       color: #fefefe;
-      @include gradient(#c3ecb4, #add39f);
+      background-color: #add39f;
+      background-image: linear-gradient(#c3ecb4, #add39f);
```

Comment:
Color literals like `#c3ecb4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#add39f` should only be used in variable declarations; they should be referred to via variable everywhere else.